<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BerlAquarium</class>
 <widget class="QDialog" name="BerlAquarium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Berlinese Aquarium</string>
  </property>
  <widget class="QLabel" name="berl_temperature">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="berl_room_temperature">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="berl_room_humidity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="berl_time">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
